.github/workflows/main.yml

```diff
@@ -121,8 +121,15 @@ jobs:
           --configuration Release
           ${{ runner.os == 'Windows' && '-p:IncludeNetCoreAppTargets=false' || '' }}
           --logger "GitHubActions;summary.includePassedTests=true;summary.includeSkippedTests=true"
+          --collect:"XPlat Code Coverage"
           --
           RunConfiguration.CollectSourceInformation=true
+          DataCollectionRunSettings.DataCollectors.DataCollector.Configuration.Format=opencover
+
+      - name: Upload coverage
+        uses: codecov/codecov-action@eaaf4bedf32dbdc6b720b63067d99c4d77d6047d # 3.1.4
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
⚠️ Seems you can remove.

Reply:
Yeah, currently it just doesn't set it seems it's undefined